src/components/WorkTitle.js

```diff
@@ -19,13 +19,76 @@ class WorkTitle extends Component {
             </div>
             <div>
                 <Container>
-                    <div className="row">
+                    {/* <h1 className='qr-sub-font '>
+                        How does ordering via QR work?
+                    </h1>
+                    <Row>
+                        <Col>
+                            <div className="col md-auto">
+                                <h4 className='qr-body text-left'>
+                                <ul class="list-styled">
+                                    <li>The guest scans the QR code on the table, chooses a dish or drink and pays for the order via the integratedorder site. When the order is delivered, the guest receives a QR voucher with which he or she can order and pay.</li>
+                                    <br/><li>The guest orders the first round through an employee. When the order is delivered, the guest receives a QR voucher with which he or she can place an order.</li>
+                                    <br/><li>The guest scans the QR code on the table, chooses a dish or drink and pays for the order via the integrated ordering site. To reorder, the guest scans the QR code on the table again and pays immediately.</li>
+                                </ul>
+                            </h4>
+                            </div>
+                        </Col>
+                        <Col>
+                            <div className="col md-auto">
+                                <Image className="img-fluid col-sm-12" src={scanQR} alt="logo"></Image>
+                            </div>
+                        </Col>
+                        
+                    </Row> */}
+                    <h1 className='qr-sub-font '>
+                        How QR ordering Works?
+                    </h1>
+                    <Row>
+                        <Col xs={12} sm={6}>
+                                <Image className="img-fluid col-sm-10" src={scanQR} alt="logo"></Image>
+                        </Col>
+                        <Col xs={12} sm={6}>
+                            <h4 className='qr-body mt-10 display-10 text-left'>
+                                <ul class="list-styled">
+                                    <li>The guest scans the QR code on the table, chooses a dish or drink and pays for the order via the integratedorder site. When the order is delivered, the guest receives a QR voucher with which he or she can order and pay.</li>
```

Comment:
Please shorten the text and remove inferred information, each in a list item/new line

1. Scan QR Code
2. Select your order
3. Pay